mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -218,4 +219,53 @@ public void reasonableLongTermShouldNotBeIgnored() throws Exception {
         assertThat(messageManager.search(new SearchQuery(SearchQuery.bodyContains(reasonableLongTerm)), session))
             .containsExactly(composedMessageId.getUid());
     }
+
+    @Test
+    @Ignore("JAMES-2078 issue with dynamic mapping")
+    public void headerSearchShouldIncludeMessageWhenDifferentTypesOnADynamicallyIndexedField() throws Exception {
```

Comment:
s/headerSearchShouldIncludeMessageWhenDifferentTypesOnADynamicallyIndexedField/headerSearchShouldIncludeMessageWhenDifferentTypesOnAnIndexedField

I think the reference to dynamic mapping has its place in the ignore message. But not on the test name, as the solution will certainly be to remove the dynamic mapping and make the test pass. No ?

Reply:
Good point thanks !